tests/src/core/RelativePathsTest.cpp

```diff
@@ -26,25 +26,36 @@
 
 #include "ConfigManager.h"
 #include "SampleBuffer.h"
+#include "PathUtil.h"
 
 #include <QDir>
 
 class RelativePathsTest : QTestSuite
 {
 	Q_OBJECT
 private slots:
-	void RelativePathComparisonTests()
+	void PathUtilComparisonTests()
 	{
 		QFileInfo fi(ConfigManager::inst()->factorySamplesDir() + "/drums/kick01.ogg");
 		QVERIFY(fi.exists());
 
 		QString absPath = fi.absoluteFilePath();
-		QString relPath = "drums/kick01.ogg";
+		QString oldRelPath = "drums/kick01.ogg";
+		QString relPath = PathUtil::basePrefix(PathUtil::FactorySample) + "drums/kick01.ogg";
```

Comment:
```suggestion
		QString relPath = PathUtil::basePrefix(PathUtil::FactorySampleBase) + "drums/kick01.ogg";
```

Reply:
Thanks! Lots of stupid mistakes in these last few commits :)